api-docs/influxdb3/core/v3/ref.yml

```diff
@@ -1280,6 +1328,50 @@ paths:
           description: Processing failure.
       tags:
         - Processing engine
+  /api/v3/configure/token/admin:
+    post:
+      operationId: PostCreateAdminToken
+      summary: Create admin token
+      description: |
+        Creates an admin token.
+        An admin token is a special type of token that has full access to all resources in the system.
+
+        This endpoint is only available in InfluxDB 3 Enterprise.
+      responses:
+        '201':
+          description: |
+            Success. The admin token has been created.
+            The response body contains the token string and metadata.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AdminTokenObject'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+      tags:
+        - Authentication
+        - Token
+  /api/v3/configure/enterprise/token/admin/regenerate:
+    post:
+      operationId: PostRegenerateAdminToken
+      summary: Regenerate admin token
+      description: |
+        Regenerates an admin token and revokes the previous token with the same name.
+
+        This endpoint is only available in InfluxDB 3 Enterprise.
```

Comment:
The token regenerate endpoint is available in Core too; is `/api/v3/configure/enterprise/token/admin/regenerate` what is used in Core or should this be updated?

Reply:
I checked these several times and still messed it up! Thanks for noticing.